docs/docsite/rst/reference_appendices/faq.rst

```diff
@@ -608,8 +608,8 @@ We also offer free web-based training classes on a regular basis. See our
 
 .. _web_interface:
 
-Is there a web interface / REST API / etc?
-++++++++++++++++++++++++++++++++++++++++++
+Is there a web interface / REST API / GUI?
+++++++++++++++++++++++++++++++++++++++++++++
```

Comment:
```suggestion
Is there a web interface / REST API / GUI?
++++++++++++++++++++++++++++++++++++++++++++
```
@acozine after i committed what you suggested, the separator line became a bit longer than the header's line, is it ok?

Reply:
Sorry, I must've counted wrong. I'll leave it - if it's too short, it generates an error; if it's too long, it works fine but looks untidy. We can tidy it up later.